core/build.gradle.kts

```diff
@@ -315,8 +317,19 @@ android {
 }
 
 androidComponents.onVariants {
-        tasks.named("preBuild") {
-            dependsOn(moveJDBCJNIFiles)
+    tasks.named("preBuild") {
+        dependsOn(moveJDBCJNIFiles)
+    }
+}
+
+buildkonfig {
+    packageName = "com.powersync.core"
+    defaultConfigs {
+        buildConfigField(STRING, "LIBRARY_VERSION", version.toString())
+
+        // TODO: Swift SDK relies on this too.
+        // Find out how to add a build flag to toggle between "powersync-kotlin" and "powersync-swift".
+        buildConfigField(STRING, "LIBRARY_NAME", "powersync-kotlin")
```

Comment:
I think we don't have to worry about this. We'll likely add an API for clients to add their own headers in the future. The Swift SDK could use that to always override the `User-Agent` header if necessary. So it should be fine if this always reports `powersync-kotlin` for now.